vmdb/app/assets/javascripts/miq_dhtmlxtoolbar.js

```diff
@@ -232,9 +232,9 @@ function miqToolbarOnClick(id) {
 
 function miqSupportCasePrompt(tb_url) {
   var support_case = prompt('Enter Support Case:', '');
-  if (support_case == null) {
+  if (!support_case) {
```

Comment:
This might be wrong.. it does change the current behavior - previusly the alert would trigger on OK but not on cancel, this way, it will not trigger on cancel or on ok with an empty string, but will trigger on ok + spacey string.